kotest-assertions/src/jvmMain/kotlin/io/kotest/Retry.kt

```diff
@@ -4,6 +4,10 @@ import io.kotest.assertions.Failures
 import kotlinx.coroutines.delay
 import java.time.Duration
 
+/**
+ * Retry the given lambda [f] in case of assertion error[AssertionError] and exception [Exception].
+ * Stops trying when [maxRetry] or [maxDelay] duration is reached.
+ * */
 suspend inline fun <T> retry(
```

Comment:
Do we want to inline it if it's suspend ?

Reply:
No i have copied code from eventually and had forgot to removed that